app/src/main/java/fr/free/nrw/commons/upload/depicts/DepictsDao.kt

```diff
@@ -0,0 +1,20 @@
+package fr.free.nrw.commons.upload.depicts
+
+import androidx.room.*
+
+@Dao
+interface DepictsDao {
```

Comment:
What is the equivalent class for categories? (the DAO that saves previously selected categories)